solutions/fully-configurable/variables.tf

```diff
@@ -262,8 +262,7 @@ variable "cloud_logs_retention_period" {
 
 variable "existing_event_notifications_instances" {
   type = list(object({
-    en_instance_id      = string
-    en_region           = string
+    en_crn              = string
```

Comment:
The markdown doc needs to update for this change right? It seems the doc is already invalid, and the link in the variable needs to be updated to point to the correct title for the `existing_event_notifications_instances` input